documentation/using/master.adoc

```diff
@@ -35,7 +35,6 @@ include::modules/proc-uninstalling.adoc[leveloffset=+1]
 
 include::modules/proc-accessing-resource-status.adoc[leveloffset=+1]
 
-include::modules/ref-kafka-logging.adoc[leveloffset=+1]
```

Comment:
Just checking that there is nothing to add to the `master.adoc` file.

Reply:
No. We just removed the appendix.